app/javascript/react/components/SessionsListView/SessionsListView.tsx

```diff
@@ -30,6 +35,41 @@ const SessionsListView: React.FC<SessionsListViewProps> = ({
   const { t } = useTranslation();
   const results = sessions.length;
   const sessionsIds = sessions.map((session) => session.id.toString());
+  const exportButtonRef = React.useRef<HTMLDivElement>(null);
+  const [buttonPosition, setButtonPosition] = React.useState({
+    top: 0,
+    left: 0,
+  });
+  const [showExportPopup, setShowExportPopup] = React.useState(false);
+  const [showAlert, setShowAlert] = React.useState(false);
+  const [alertMessage, setAlertMessage] = React.useState<string | null>(null);
+
+  const rect = exportButtonRef.current?.getBoundingClientRect();
+  const NO_SESSIONS = sessionsIds.length === 0;
+  const EXCEEDS_LIMIT = sessionsIds.length > SESSIONS_LIMIT;
+  const popupTopOffset = NO_SESSIONS ? -13 : -50;
+
+  const updateButtonPosition = () => {
+    if (rect) {
+      setButtonPosition({
+        top: rect.top + window.scrollY,
+        left: rect.left + window.scrollX,
+      });
+    }
+  };
+
+  useEffect(() => {
+    updateButtonPosition();
+    window.addEventListener("resize", updateButtonPosition);
+
+    return () => {
+      window.removeEventListener("resize", updateButtonPosition);
+    };
+  }, [rect?.top]);
```

Comment:
here you can just rely on updateButtonPosition in dependency array, or is it causing some issues?